asset/src/__lib/chunked-file-reader/formatters.ts

```diff
@@ -136,9 +139,11 @@ function json(incomingData, logger, opConfig, metadata) {
     }
 }
 
-function ldjson(incomingData, logger, opConfig, metadata, slice) {
+export function ldjson(
+    incomingData: any, logger: Logger, opConfig: any, metadata: any, slice: any
+) {
     const data = _toRecords(incomingData, opConfig.line_delimiter, slice);
-    return data.map((record) => {
+    return data.map((record: any) => {
```

Comment:
Adding this `(record: any)`  isn't needed if proper types are returned from `_toRecords`